### PR TITLE
Feat/224 connect with arconnect

### PR DESCRIPTION
**Pull request by jdaev**

### CODE REVIEW

**Review by fedellen — APPROVED** [review]
Looks good 👍

**Comment by arielmelendez** [comment]
@jdaev , is there anything in this PR that wasn't present in all the other PRs that the team has reviewed?

**Comment by jdaev** [comment]
> @jdaev , is there anything in this PR that wasn't present in all the other PRs that the team has reviewed?

No, it's just for code review.

File: lib/blocs/drive_create/drive_create_cubit.dart [hunk 1]
```diff
@@ -89,15 +98,16 @@ class DriveCreateCubit extends Cubit<DriveCreateState> {
...
-        wallet,
+        profile.getRawWalletSignature,
+        owner,
         createRes.driveKey,
       );
 
       await _arweave.postTx(driveTx);
       await _arweave.postTx(rootFolderTx);
-
+      print(driveTx.id);
```
Comment (arielmelendez) [inline lib/blocs/drive_create/drive_create_cubit.dart:108]: Unnecessary prints.

File: lib/blocs/drive_detail/drive_detail_cubit.dart [hunk 2]
```diff
@@ -80,12 +80,12 @@ class DriveDetailCubit extends Cubit<DriveDetailState> {
               ? this.state as DriveDetailLoadSuccess
               : DriveDetailLoadSuccess();
           final profile = _profileCubit.state;
-
+          //TODO: Why use wallet.getAddress() here instead of profile.walletAddress?
```
Comment (arielmelendez) [inline lib/blocs/drive_detail/drive_detail_cubit.dart:83]: Does this need to followed up on?

File: lib/blocs/drives/drives_cubit.dart [hunk 3]
```diff
@@ -45,9 +45,10 @@ class DrivesCubit extends Cubit<DrivesState> {
       }
 
       final profile = _profileCubit.state;
+      //TODO: Again why is wallet.getAddress() used instead of walletAddress
       final walletAddress =
-          profile is ProfileLoggedIn ? await profile.wallet.getAddress() : '';
-
+          profile is ProfileLoggedIn ? await profile.walletAddress : '';
+      //TODO: Some way to tell if the owner of the wallet is logged in?
```
Comment (arielmelendez) [inline lib/blocs/drives/drives_cubit.dart:51]: Does this also need to be followed up on?

File: lib/blocs/drive_detail/drive_detail_cubit.dart (hunk 2 above)
Comment (jdaev) [reply lib/blocs/drive_detail/drive_detail_cubit.dart:83]: No, it was just something I was confused about when I started working on this. Will remove.

File: lib/blocs/drives/drives_cubit.dart (hunk 3 above)
Comment (jdaev) [reply lib/blocs/drives/drives_cubit.dart:51]: Same as above. Will remove.

**Review by arielmelendez — APPROVED** [review]
Just double check the last few questions I left about the TODOs that were scattered around.

File: lib/blocs/drives/drives_cubit.dart (hunk 3 above)
Comment (jdaev) [reply lib/blocs/drives/drives_cubit.dart]: [Removed](https://github.com/ardriveapp/ardrive-web/pull/124/commits/320a20c9ce8279d1c9163cf61a53e8d574904bcd)

File: lib/blocs/drive_detail/drive_detail_cubit.dart (hunk 2 above)
Comment (jdaev) [reply lib/blocs/drive_detail/drive_detail_cubit.dart]: [Removed ](https://github.com/ardriveapp/ardrive-web/pull/124/commits/320a20c9ce8279d1c9163cf61a53e8d574904bcd)

File: lib/blocs/drive_create/drive_create_cubit.dart (hunk 1 above)
Comment (jdaev) [reply lib/blocs/drive_create/drive_create_cubit.dart]: [Removed ](https://github.com/ardriveapp/ardrive-web/pull/124/commits/320a20c9ce8279d1c9163cf61a53e8d574904bcd)

**Pull request merged**